primitives/core/src/testing.rs

```diff
@@ -152,3 +152,13 @@ impl crate::traits::SpawnNamed for TaskExecutor {
 		self.0.spawn_ok(future);
 	}
 }
+
+#[cfg(feature = "std")]
+impl crate::traits::SpawnEssentialNamed for TaskExecutor {
```

Comment:
where is this used? and why is the name not used?

Reply:
As the file name indicates, this is only for testing purposes :D